### PR TITLE
Fixes stop pulling keybind

### PR DESCRIPTION
**Pull request by DominikPanic**
Fixes: #1926

### CODE REVIEW

File: code/datums/keybinding/mob.dm [hunk 1]
```diff
@@ -57,7 +57,7 @@
 
 /datum/keybinding/mob/stop_pulling/down(client/user)
 	var/mob/M = user.mob
-	if (M.pulling)
+	if(!M.pulling)
```
Comment (psykzz) [inline code/datums/keybinding/mob.dm]:
```suggestion
	if(M.pulling)
	    M.stop_pulling()
```

Just delete the rest, I don't feel the message is helpful on keybindings.

Comment (Rohesie) [reply code/datums/keybinding/mob.dm]: Any chance of making this a signal? I can guide you through how, and it's useful knowledge.

Comment (DominikPanic) [reply code/datums/keybinding/mob.dm]: reeee sure

Comment (spookydonut) [reply code/datums/keybinding/mob.dm]: doesnt need to be a signal, pulling and stop_pulling are defined at mob or above

Comment (spookydonut) [reply code/datums/keybinding/mob.dm]: just do the suggested change and it can merge

**Pull request merged**